branch-name-content-script.js

```diff
@@ -23,3 +24,37 @@ function getBranchName() {
     .toLowerCase()
     .replace(/\s+/g, "-");
 }
+
+const BUTTON_ID = "get-branch-name-button";
+
+function addBranchNameButton() {
+  if (document.getElementById(BUTTON_ID)) return;
+  const githubHeader = document.getElementsByClassName("gh-header-actions")[0];
+  if (!githubHeader) return;
+
+  var getBranchNameButton = document.createElement("button");
```

Comment:
The `get` prefix tends to denote a helper function. How about going with `branchNameButton` for easier readability?

Reply:
Hmm, oh I think I named it this because it's a button that "gets the branch name" but that is def confusing.